saleor/app/management/commands/create_app.py

```diff
@@ -13,6 +14,8 @@
 from ...models import App
 from .utils import clean_permissions
 
+REQUEST_TIMEOUT = (settings.REQUESTS_CONN_EST_TIMEOUT, 15)
```

Comment:
We talked about it with @patrys that at this point we could set all timeouts to the same value and have 20s total, but split as follows:
- connection timeout - 2s
- read timeout - 18s

Reply:
Yes, I suggest we set the default timeout of hardened requests to (2, 18) and not provide any overrides unless strictly required (and those cases need a big fat comment explaining why it needs to be special).

Reply:
I believe there is a PR to add https://github.com/saleor/requests-hardened to our stack.

Reply:
> I believe there is a PR to add https://github.com/saleor/requests-hardened to our stack.

Added in 3.16.0 via #13891 


You can override the global default here: https://github.com/saleor/saleor/blob/cc0ff0f7a533ddf5fb2a0ad4e500fa5c5e3b88ea/saleor/core/http_client.py#L8?0, when `HTTPClient.send_request()` doesn't define `timeout=...` (thus `timeout` is `None`) then it will fallback to `default_timeout`.

I recommend to define multiple clients if you want to split timeouts between kinds, e.g.:
- `AsyncHTTPClient = requests_hardened.Manager(...)`
- `SyncHTTPClient = requests_hardened.Manager(...)`

Then in the code-base you can use the adequate client for the use case.